docs/antora-ccutil/ccutil.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+rm -rf target/antora/build/ccutil
+mkdir -p target/antora/build/ccutil
+
```

Comment:
ccutil.sh failed for me as I didn't have "PROJECT_VERSION" set. This will set it to the version from the POM if not already set.

```suggestion

if [ "$PROJECT_VERSION" == "" ]; then
  PROJECT_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
fi

```

Reply:
Replaced the only usage in the file to "target/keycloak-antora-ccutil-*-ccutil.zip", no ENV var needed any more here.